apps/synapse-portal-framework/src/components/RedirectToURL.tsx

```diff
@@ -21,4 +21,5 @@ export default function RedirectToURL(props: RedirectToURLProps) {
   if (isSearchMatch) {
     window.location.replace(toURL)
   }
+  return null
```

Comment:
This function is typically used as a component. For type-safety, it should explicitly return.